api/tests/unit/cat/skill_test.js

```diff
@@ -55,13 +55,46 @@ describe('Unit | Model | Skill', function() {
 
     it('should return url3 and url5 when requesting skills harder than url3 within url1-3-5', function() {
       // given
-      const url1 = new Skill('url1');
-      const url3 = new Skill('url3');
-      const url5 = new Skill('url5');
+      const url1 = new CatSkill('url1');
+      const url3 = new CatSkill('url3');
+      const url5 = new CatSkill('url5');
       const tubes = { 'url': [url1, url3, url5] };
 
       // then
       expect(url3.getHarderWithin(tubes)).to.be.deep.equal([url3, url5]);
     });
   });
+
+  describe('Skill.compare()', function() {
+    it('should take two skills and compare them', () => {
```

Comment:
Est-ce qu'on pourrait renommer le test pour décrire le comportement attendu (par exemple "should return false if the skills have different names")

Reply:
👍